views/v_ksm_alternate_regions.sql

```diff
@@ -0,0 +1,47 @@
+--- Goal: Inviting prospects to cultivation events in other cities in addition to their preferred home/work region.
+--- We will utilize existing Kellogg Alumni Club committee codes to track alternate regions.
+--- *** Code will be dependant on Advancement's entry of comment code, KARP in committees. ***
+
+Create or Replace View v_ksm_alternate_regions AS
+
+With alternate_region As (Select comm.id_number,
+       comm.stop_dt,
+       comm.committee_code,
+       tms.short_desc,
+       comm.start_dt,
+       comm.committee_status_code,
+       comm.committee_title,
+       comm.geo_code,
+       comm.xcomment
+FROM  committee comm
+Inner Join rpt_pbh634.v_entity_ksm_degrees deg on deg.ID_NUMBER = comm.id_number
```

Comment:
This view isn't used in the select; remove for speed

Reply:
Removed v_entity_ksm_degrees